src/tests/networking_tests.rs

```diff
@@ -1544,6 +1556,167 @@ pub mod net_tests {
         lindrustfinalize();
     }
 
+    pub fn ut_lind_net_socketpair_bad_input() {
+        // test for error cases of socketpair
+        lindrustinit(0);
+        let cage = interface::cagetable_getref(1);
+        let mut socketpair = interface::SockPair::default();
+
+        // test for unspported domain
```

Comment:
please state the unsupported domains explicitly

also, lets correct the typo to "unsupported" throughout